kinto/tests/test_views_collections.py

```diff
@@ -0,0 +1,40 @@
+from .support import BaseWebTest, unittest
+from base64 import b64encode
+
+
+def get_user_headers(user):
+    return {
+        'Authorization': 'Basic {0}'.format(b64encode("%s:secret" % user)),
+    }
+
+
+class CollectionViewTest(BaseWebTest, unittest.TestCase):
+    def test_empty_collection_returns_an_empty_list(self):
+        response = self.app.get('/collections/barley/records',
+                                headers=self.headers)
+        self.assertEqual(response.json['items'], [])
+
+    def test_individual_collections_can_be_deleted(self):
+        self.app.post('/collections/barley/records',
+                      headers=self.headers)
+
+        self.app.delete('/collections/barley/records',
+                        headers=self.headers)
+
+    def test_items_can_be_added_to_collections(self):
+        response = self.app.post('/collections/barley/records',
```

Comment:
I'm updating the tests to use a non-empty record. 

I'm actually not sure we should support empty records, but I don't see any good reason to not support them.